gradle/publish-dists.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'maven-publish'
```

Comment:
I would be surprised if this works.  It doesn't appear to have a publishing block that would package up the distTar.  It has a publishing task but that doesn't seem to have an artifact it's publishing...